### DISCUSSION TITLE
Submit your results

### DISCUSSION THREAD

**Discussion by mrdbourke** [mrdbourke/m1-machine-learning-test#1] [General]
If you end up running the tests on your own machine, feel free to leave your results here so others can see.

Feel free to use this template:

```markdown
| Machine | Chip | Dataset | Model | Num Epochs | Time per epoch (average, seconds) |
| ----- | ----- | ----- | ----- | ----- | ----- |
|  |  |  |  |  |  |
```

Post the above with your results in between the pillars on the bottom for each column.

Like this:

```markdown
| Machine | Chip | Dataset | Model | Num Epochs | Time per epoch (average, seconds) |
| ----- | ----- | ----- | ----- | ----- | ----- |
| 2021 Apple 14-inch MacBook Pro  |  M1 Pro  | CIFAR10 | TinyVGG | 10  | 17 |
```

**Comment by lexeremin** [comment]
Hey! Here are some of the results that i've done on my machine:
| Machine | Chip | Dataset | Model | Num Epochs | Time per epoch (average, seconds) |
| ----- | ----- | ----- | ----- | ----- | ----- |
|  PC Ryzen 5600x 16GB RAM Windows 11  |  GTX 1080  |  CIFAR10  |  TinyVGG  |  10  |  7.8  |
|  PC Ryzen 5600x 16GB RAM Windows 11  |  GTX 1080  |  CALIFORNIA_HOUSING  |  RandomForestCV  |  -  |  18.7  |
|  PC Ryzen 5600x 16GB RAM Windows 11  |  GTX 1080  |  FOOD101  |  EfficientNetB0  |  5  |  171.2  |

**Comment by kartikra** [comment]
Added my results of testing using 3080 GPU on Lenovo Tested on ubuntu 22.04
 https://github.com/kartikra/m1-machine-learning-test/tree/3080-laptop-test/results

**Comment by DennisFaucher** [comment]
| Machine | Chip | Dataset | Model | Num Epochs | Time per epoch (average, seconds) |
| Apple Mac Mini M1 | Apple M1 | CIFAR10 | TinyVGG | 10 | 29.335 |

**Comment by DennisFaucher** [comment]
| Machine | Chip | Dataset | Model | Num Epochs | Time per epoch (average, seconds) |
| Apple Mac Mini M1 | Apple M1 | California Housing | RandomForestCV | N/A | 64.356 |

**Comment by DennisFaucher** [comment]
| Machine | Chip | Dataset | Model | Num Epochs | Time per epoch (average, seconds) |
| Apple Mac Mini M1 | Apple M1 | Food 101 | EfficientNetB0 | 5 | 2878.399 |
